pkg/controller/master/node/promote_controller.go

```diff
@@ -34,6 +34,8 @@ const (
 	HarvesterPromoteNodeLabelKey        = HarvesterLabelAnnotationPrefix + "promote-node"
 	HarvesterPromoteStatusAnnotationKey = HarvesterLabelAnnotationPrefix + "promote-status"
 
+	HarvesterTopologyZoneLabel = "topology.kubernetes.io/zone"
```

Comment:
can be replaced by https://github.com/harvester/harvester/blob/ff26eab62ad3b1799ee5f4e713a20410c0fb15a1/vendor/k8s.io/api/core/v1/well_known_labels.go#L22